cider-velox/src/TypeConversions.h

```diff
@@ -84,8 +86,20 @@ inline TypePtr getVeloxType(::substrait::Type& typeName) {
       return TIMESTAMP();
     case ::substrait::Type::KindCase::kDate:
       return DATE();
+    case ::substrait::Type::KindCase::kStruct: {
+      std::vector<TypePtr> rowTypes;
+      std::vector<std::string> names;
```

Comment:
reserve the size for rowTypes and names as  struct type size

Reply:
done.